torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -86,12 +86,18 @@ static void getMajorMinor(
     max_dev_version = CudaVersion(5, 0);
   } else if (nvrtc_version.first <= 8) { // 8 supports 2-6.x
     max_dev_version = CudaVersion(6, 0);
-  } else if (nvrtc_version.first <= 9) { // 9 supports 3-7.2
+  } else if (
+      nvrtc_version.first <= 9 ||
```

Comment:
remove changes to this file, this was already fixed in #48151.

Reply:
This is what I was worried about, the fix in #48151 assumes cuda 11.0 supports compute_80. Which contradicts NVRTC doc:
https://github.com/pytorch/pytorch/pull/48151/files#diff-a0854dccd6b375eb09b2a882f208b30719ca720bb06028b0dd4500d30d7aae23R94